coalib/bearlib/aspects/meta.py

```diff
@@ -72,3 +75,24 @@ class Sub(subcls, aspectbase, metaclass=aspectclass):
 
     def __repr__(cls):
         return '<%s %s>' % (type(cls).__name__, repr(cls.__qualname__))
+
+
+def issubaspect(aspect, subaspect):
+    """
+    This function checks whether or not ``subcls`` is a subaspect of cls
```

Comment:
If the arguments passed here are `aspect` and `subaspect`, shouldn't it be better if they are used in the docstring instead of `subcls` and `cls`